### DISCUSSION TITLE
OS/

### DISCUSSION THREAD

**Discussion by giscus** [kuaimiaoda/Giscus_Comment#1] [Announcements]
# OS/

操作系统期末总结第一章 操作系统概述一、基本概念1.操作系统：是一组能有效组织和管理计算机硬件和软件资源，合理地对各类作业进行调度，以及方便用户使用的程序（软件）的集合。 2.并发性：提高资源利用率、增加系统的吞吐量。 3.异步性：每个程序在何时执行，多个程序间的执行顺序以及完成每道程序所需的时间都是不确定的，因而也是不可预知的，即进程以不可预知的速度向前推进 ——由于资源等因素的限制，使进程的执

http://localhost:4000/OS/

**Comment by kuaimiaoda** [comment]
123
